docs/product-assembly_pam-on-openshift/src/main/asciidoc/environment-authoring-ha-modify-proc.adoc

```diff
@@ -0,0 +1,45 @@
+[id='environment-authoring-ha-modify-proc']
+= Modifying the template for the High Availability authoring environment
```

Comment:
Side note: The title still does not have the complete OpenShift name (Red Hat OpenShift Container Platform). I mentioned in my last review to be sure to do that, via an attribute. This is important mainly because starting today, I'm creating cross-reference links in document-attributes for all new assemblies, and am creating the doc titles in Pantheon for the publish. Doc titles need to be correct, exact, and finalized for everything to work properly. I sent an email to the team yesterday (Monday) reminding everyone of this. Please make sure all your OpenShift docs use the full OpenShift name. Again, I recommend an attribute. If you have a specific reason for not doing so, let me know.

Reply:
Please ignore this issue in this assembly. This particular assembly is not being built for production, anyway. The decision was taken to split OpenShift into three documents and I have made three new assemblies, which are now in first QA review (without modifying templates); when I complete Modifying I hope to send the documents for your quick peer review. 

So this assembly just holds the files and its main.adoc is for internal builds - in this build I stripped it to one topic.

I have not moved the topics into a guide yet because that would destroy the possibility of global searches and make things too hard to edit. We can do that sometime after GA when under less pressure.